src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -63,8 +63,7 @@ public void periodic() {
 		double joyX = RobotContainer.driverController.getRawAxis(OperatorConstants.XBOX_LEFT_X_AXIS);
 		double joyY = -RobotContainer.driverController.getRawAxis(OperatorConstants.XBOX_LEFT_Y_AXIS);
 		double rotation = ControlConstants.ROTATION_MULT
-				* (RobotContainer.driverController.getRightTriggerAxis() - RobotContainer.driverController.getLeftTriggerAxis());
-
+				* (RobotContainer.driverController.getRawAxis(OperatorConstants.XBOX_RIGHT_X_AXIS) - RobotContainer.driverController.getRawAxis(OperatorConstants.XBOX.RIGHT_Y_AXIS));
```

Comment:
we JUST want the horizontal X axis of the right stick - don't subtract anything from it. The X axis value will return both a positive and a negative